UML editor/src/main/java/codecain/UMLFieldInfo.java

```diff
@@ -0,0 +1,75 @@
+package codecain;
+
+/**
+ * The UMLFieldsInfo class represents a field in a UML class.
+ * It includes the field name and the field type.
+ */
+public class UMLFieldInfo {
```

Comment:
UMLFieldInfo is a nice addition we didn't have in sprint1. Having multiple maps last sprint had me confused about what exactly was going on, and having this clarity will nice for everyone